packages/embarkjs/src/utils.js

```diff
@@ -1,4 +1,6 @@
-/* global Web3 clearInterval setInterval */
+/* global clearInterval global require setInterval */
+
+const Web3 = global.Web3 || require('web3');
```

Comment:
Is this needed? Doesn't look like there's another change in this file that actually uses `Web3`...

Reply:
See: https://github.com/embark-framework/embark/blob/9d9001476b578d9656469611304225119cfad630/packages/embarkjs/src/utils.js#L7.

The methods defined in that module make reference to a global `Web3`. Generally that's bad, but can work out if some other code has already set a global `Web3`. This change makes sure that if global `Web3` isn't available the script can still be loaded directly and its methods are actually usable.